app/views/components/docs/share-links.yml

```diff
@@ -0,0 +1,34 @@
+name: Share links
+description: Links to allow the user to share the current page on Facebook or Twitter
+body: |
+  Pass complete share URls to the component - the component will not process a URL into a share link itself.
+
+  The component also  contains Javascript classes (js-share-facebook and js-share-twitter) which are used to track the number of clicks on the share links.
```

Comment:
I'd rephrase this slightly:
The component will track interactions with the share links to Google Analytics using [Social Interactions](https://developers.google.com/analytics/devguides/collection/analyticsjs/social-interactions)